lib/http.ex

```diff
@@ -71,20 +71,40 @@ defmodule Braintree.HTTP do
   """
   @spec request(atom, binary, binary | map, Keyword.t()) :: response
   def request(method, path, body \\ %{}, opts \\ []) do
-    response =
+    emit_start(method, path)
+
+    start_time = System.monotonic_time()
+
+    try do
       :hackney.request(
         method,
         build_url(path, opts),
         build_headers(opts),
         encode_body(body),
         build_options()
       )
-
-    case response do
+    catch
+      kind, reason ->
+        stacktrace = System.stacktrace()
```

Comment:
This is deprecated, we need to use `__STACKTRACE__` instead. If the minimum elixir version is too low we can bump it as well.